lib/query/query_supporters.rb

```diff
@@ -506,47 +506,48 @@ def self.dupes_on_email(np_id)
     dupes_expr(np_id)
       .and_where("email IS NOT NULL")
       .and_where("email != ''")
-      .group_by(:email)
+      .group_by("btrim(lower(email, ' '))")
```

Comment:
I don't think it's wrong but this is a change of behavior. Do we know that behavior is safe?

Reply:
As emails are not case-sensitive and spaces are not valid characters for emails, I would say it is safe. Do you see any problems with our database that could be a problem for this?